Support bytes concat (#14685)
ethereum/solidity: Changelog.md

Compiler Features:

> * SMTChecker: Support `bytes.concat` function with an exception of string literals provided as an argument to `bytes.concat` call.

Reviewer: ```suggestion
* SMTChecker: Support `bytes.concat` except when string literals are passed as arguments.
```
